python/create_client/create_client.py

```diff
@@ -14,6 +14,7 @@
 
 # Coming from the vendors folder
 import websocket
+import six
```

Comment:
Not importing from tank_vendor, as this code can be used standalone.

Reply:
I actually went back on this, and did import from tank_vendor if possible, so as to try and avoid and clashes with previously imported six modules that might be old.
It now falls back to importing just six if it can't from tank_vendor.